messages.go

```diff
@@ -268,3 +268,46 @@ func (receipt *msgReceipt) PublishExecutionError(err string, trace []string) err
 		},
 	)
 }
+
+const (
+	// StreamStdout defines the stream name for standard out on the front-end. It
+	// is used in `PublishWriteStream` to specify the stream to write to.
+	StreamStdout = "stdout"
```

Comment:
These should be defined in `kernel.go` because that is where they are referenced.

Reply:
I think that because the values are protocol magic values that they should really be defined in the messages file.

Reply:
Yeah, I think I just said this because I was looking for what they were in the `kernel.go` file and couldn't find them. We can leave them for now though, as it's not a huge deal and if you feel strongly here.